crates/sui-framework/src/natives/crypto.rs

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+use move_binary_format::errors::PartialVMResult;
+use move_vm_runtime::native_functions::NativeContext;
+use move_vm_types::{
+    gas_schedule::NativeCostIndex,
+    loaded_data::runtime_types::Type,
+    natives::function::{native_gas, NativeResult},
+    pop_arg,
+    values::Value,
+};
+use narwhal_crypto::traits::ToFromBytes;
+use smallvec::smallvec;
+use std::collections::VecDeque;
+
+pub fn ecrecover(
```

Comment:
Add a couple of line of docs (what this fun does for ECDSA) as this is public api.